components/global/DrupalForm.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <form vue-enabled :formId="formId" :method="method" v-bind="attributes" @submit.prevent="submitForm">
```

Comment:
this submits the form with JS. we want form-submissions to work progressively, thus we need to make it work without JS first.